src/taxes/components/TaxAppLabel.tsx

```diff
@@ -68,12 +75,11 @@ export const TaxAppLabel: React.FC<TaxAppLabelProps> = ({
       </Box>
       <Box
         as="a"
```

Comment:
suggestion: This is now an invalid link so, let's better use `button` instead:

```suggestion
        as="button"
```

Reply:
I updated to be a button from new MacawUI